### PR TITLE
When the Copy Pasta Monster Fails...

### PR DESCRIPTION
**Pull request by Alexshreds**
Fixes: #5114
Fixes my sanity
Fixes bad copy pasta
<s>Actually HTML has +20 INT stat compared to BYOND so it's smart enough to not have a seizure over multiple tabs, it's just for meh OCD.</s>

:cl: Alexshreds
Tweak: pAIs can see the crew manifest again.
/:cl:

### CODE REVIEW

File: nano/templates/pai_manifest.tmpl [hunk 1]
```diff
@@ -2,97 +2,99 @@
 code/modules/mob/living/silicon/pai/software_modules.dm
 
 Copy-pasted from pda.tmpl
+
+Alexshreds - Whoever originally copy-pasted this needs to work on their copy-pasta skills plz.
```
Comment (Fox-McCloud) [inline nano/templates/pai_manifest.tmpl]: Please leave this comment out.

Comment (Iamgoofball) [inline nano/templates/pai_manifest.tmpl]: no

Comment (GeneralGoose) [inline nano/templates/pai_manifest.tmpl]: people need to know their heroes

**Comment by monster860** [comment]
Oh great, pAI's will now be in charge of spamming suit sensor reminders. And there will be *nothing* the crew will be able to do about it

**Comment by Spacemanspark** [comment]
Crew manifest.
Not crew monitoring console.

**Comment by monster860** [comment]
Oh

**Pull request merged**